net-libs/libhubbub/libhubbub-0.3.6.ebuild

```diff
@@ -0,0 +1,53 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit toolchain-funcs flag-o-matic
+
+DESCRIPTION="HTML5 compliant parsing library, written in C"
+HOMEPAGE="http://www.netsurf-browser.org/projects/hubbub/"
+SRC_URI="https://download.netsurf-browser.org/libs/releases/${P}-src.tar.gz"
+
+LICENSE="MIT"
+SLOT="0/${PV}"
+KEYWORDS="~amd64 ~arm ~ppc ~ppc64 ~x86 ~m68k-mint"
+IUSE="doc test"
+
+RDEPEND="
+	>=dev-libs/libparserutils-0.2.1-r1
+	!net-libs/hubbub"
+DEPEND="${RDEPEND}
+	dev-util/netsurf-buildsystem
+	virtual/pkgconfig
+	test? (
+		dev-lang/perl
+		<dev-libs/json-c-0.13
```

Comment:
Are you sure about this requirement? Did you check if it was updated? 
(Caused me some downgrades)

Reply:
Yes but the fix was easy. It's included as a patch now. I'll send it upstream.

Reply:
If it gets executed during the build, its a BDEPEND. If it has libraries/headers that are needed during build, its a DEPEND. Granted I dont know what this package does exactly, but looked like it possibly had both.